cachito/workers/config.py

```diff
@@ -140,6 +185,16 @@ class DevelopmentConfig(Config):
     }
     cachito_request_file_logs_dir: Optional[str] = "/var/log/cachito/requests"
     cachito_sources_dir = os.path.join(ARCHIVES_VOLUME, "sources")
+    if not os.environ.get("CACHITO_OTLP_EXPORTER_ENDPOINT"):
+        if os.environ.get("CACHITO_JAEGER_EXPORTER_ENDPOINT"):
+            print("Jaeger endpoint defined")
+            cachito_jaeger_exporter_endpoint = os.environ.get("CACHITO_JAEGER_EXPORTER_ENDPOINT")
+            cachito_jaeger_exporter_port = cast(int, os.environ.get("CACHITO_JAEGER_EXPORTER_PORT"))
```

Comment:
`cast()` is a type-checker assertion, it doesn't do anything at runtime

if the port really does need to be an int, you'll need to convert it with `int()` (but only if the environment variable is set, otherwise that's a `TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'`

Reply:
Ah, it gets converted later in

```python
        jaeger_exporter = JaegerExporter(
            agent_host_name=config.cachito_jaeger_exporter_endpoint,
            agent_port=int(config.cachito_jaeger_exporter_port),
        )
```

IMO that's the right place to do the conversion, and it should be a string in the Config class. (If we're using env vars to set the config. If we set the variable directly in the code, it can be an int)